tests/dart/DartLintBearTest.py

```diff
@@ -30,3 +37,23 @@
                                      valid_files=(good_file,),
                                      invalid_files=(bad_file,),
                                      tempfile_kwargs={"suffix": ".dart"})
+
+
+@skipIf(which('dartanalyzer') is None, 'dartanalyzer is not installed')
```

Comment:
Don't write your custom check please !
Use the skip generator for the bear ? https://github.com/coala/coala-bears/blob/master/tests/BearTestHelper.py#L4